tests/test_dse.py

```diff
@@ -625,7 +625,7 @@ def test_tti_rewrite_aggressive_wmpi():
 
 @switchconfig(profiling='advanced')
 @pytest.mark.parametrize('space_order,expected', [
-    (8, 152), (16, 268)
+    (8, 173), (16, 309)
```

Comment:
my old comments gone lost. What the hell is going on here?

Reply:
Don't know, opcount changed maybe due to FD changes